static/js/stream_edit.js

```diff
@@ -58,6 +58,9 @@ exports.hide_sub_settings = function (sub) {
 };
 
 exports.show_sub_settings = function (sub) {
+    if (!exports.is_sub_settings_active(sub)) {
+        return;
+    }
```

Comment:
@YJDave can you explain why this fix is correct in more detail?  I feel like based on the name, `exports.show_sub_settings` here feels like something that should be used to display the target tab, though the code doesn't seem to match that use case ... what is this function actually for?

Reply:
This function is used to show subscription related settings in stream setting page, i.e. notifications, email address etc. So when current user subscribed to stream we show/slides in these all settings. This can return error if there is currently no stream setting page open. In that case we don't need to render anything so just return.